libpldmresponder/platform_numeric_effecter.hpp

```diff
@@ -333,6 +336,162 @@ int setNumericEffecterValueHandler(const DBusInterface& dBusIntf,
     return PLDM_SUCCESS;
 }
 
+int getEffecterValue(std::string propertyType,
+                     pldm::utils::PropertyValue propertyValue,
+                     uint8_t* effecterValue, size_t effecterValueLength)
+{
+    if (propertyType == "uint32_t")
+    {
+        uint32_t propVal = std::get<uint32_t>(propertyValue);
+#ifdef __STDC_LIB_EXT1__
```

Comment:
why a C style copying, can't we use casting and then return the value?

Reply:
Please try of std::copy_n works here instead of memcpy

Reply:
please respond to the above comments

Reply:
Removed memcpy 

> why a C style copying, can't we use casting and then return the value?

Removed the memcpy and directly passing the effecter value by typecasting it to reinterpret_cast.

Reply:
> please respond to the above comments

I have reworked on the code flow for GetNumericEffecter. Now we are not using memcpy. Requesting review again.